app/src/main/res/layout/add_contact_dialog.xml

```diff
@@ -5,105 +5,87 @@
     android:layout_height="match_parent">
 
     <com.google.android.material.textfield.TextInputLayout
-        android:id="@+id/contact_name_layout"
+        android:id="@+id/tilAddContactName"
+        style="@style/TextInputLayoutDialog"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:textColorHint="@color/additional_text_first"
-        app:boxBackgroundColor="@color/additional_first"
-        app:cursorColor="@color/additional_text_first"
-        app:hintTextColor="@color/additional_text_first"
         app:errorEnabled="true"
         app:errorTextAppearance="@style/error_text_style"
-        app:layout_constraintWidth_percent="@dimen/user_inputs_width"
+        app:layout_constraintBottom_toTopOf="@id/tilAddContactCareer"
+        app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHeight_percent="@dimen/email_input_height"
-        app:layout_constraintVertical_bias="@dimen/email_input_vertical"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintBottom_toTopOf="@+id/contact_career_layout" >
+        app:layout_constraintVertical_bias="@dimen/email_input_vertical"
+        app:layout_constraintWidth_percent="@dimen/user_inputs_width">
```

Comment:
There is no strict requirement to extract all values to @dimen, especially when they are not used in other files.

Reply:
but it is ok